recipe/common.php

```diff
@@ -18,13 +18,32 @@
 set('clear_paths', []);         // Relative path from deploy_path
 set('clear_use_sudo', true);    // Using sudo in clean commands?
 
+
+/**
+ * Composer options
+ */
+env('composer_action', function () {
+    return env('env') === 'prod' ? 'install' : 'update';
```

Comment:
It seems needlessly restrictive to hard-code the "install" action. Would you be more content to reverse this logic such that update is only called if the environment is set to dev?

`return env('env') === 'dev' ? 'update' : 'install';`

The update command is a valid use case for deployer. For example, I am setting up a project to automatically deploy to a staging server after a successful travis build automatically. Regardless, the default is still to use `install` unless someone changes their environment.